pkgs/top-level/all-packages.nix

```diff
@@ -1882,6 +1882,10 @@ with pkgs;
 
   gopacked = callPackage ../applications/misc/gopacked { };
 
+  gotktrix = callPackage ../applications/networking/instant-messengers/gotktrix {
+    buildGoModule = buildGo117Module;
```

Comment:
Isn't the `buildGoModule` like… unnecessary?

Reply:
I just realized that `buildGoModule` is now `buildGo117Module`.